easybuild/easyconfigs/s/SRAdownloader/SRAdownloader-3.9-GCCcore-11.3.0.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'sradownloader'
+version = '3.9'
+
+homepage = "https://github.com/s-andrews/sradownloader"
+description = """SRAdownloader takes the annotation table from the SRA run selector tool 
+and retrives the raw fastq files for the selected samples"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/s-andrews/sradownloader/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dee44a8476fc11d3306019fe608295fc3b19f0c0073061ef71f1c0101dc587cc']
+
+dependencies = [
+    ('Python', '3.10.4'),
+]
+
+modextrapaths = {
+    'PATH': '',
+}
+
+sanity_check_paths = {
+    'files': ['sradownloader'],
+    'dirs': [],
+}
+
+sanity_check_commands = [
+    'sradownloader -h',
+    'sradownloader --version'
+]
+
+moduleclass = 'bio'
```

the CI is failing because there is no newline at the end of the file
```suggestion
moduleclass = 'bio'
```